app/src/main/java/org/oppia/android/app/devoptions/forcenetworktype/NetworkTypeItemViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.android.app.devoptions.forcenetworktype
+
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import org.oppia.android.util.networking.NetworkConnectionUtil
+import javax.inject.Inject
+
+/** [ViewModel] for displaying a network type for the recyclerview in [ForceNetworkTypeFragment]. */
+class NetworkTypeItemViewModel @Inject constructor(
+  val networkType: NetworkConnectionUtil.ConnectionStatus
+) : ObservableViewModel() {
+
+  /** Returns the network type in a human readable format. */
+  fun formatNetworkTypeString(): String = networkType.name.toLowerCase().capitalize()
```

Comment:
If this is being displayed to the user it ought to use strings instead of formatting the enum constant (to facilitate things like translations & to avoid refactors/enum renames changing the UI).

Reply:
I have modified the approach to use strings instead of enum names. PTAL. Thanks.

Reply:
This looks much better--thanks!